DigitalLearningSolutions.Data/Services/DelegateDownloadFileService.cs

```diff
@@ -74,7 +146,154 @@ private void PopulateJobGroupsSheet(IXLWorkbook workbook)
                     item => new { JobGroupID = item.id, JobGroupName = item.name }
                 );
 
-           ClosedXmlHelper.AddSheetToWorkbook(workbook, JobGroupsSheetName, jobGroups, TableTheme);
+            ClosedXmlHelper.AddSheetToWorkbook(workbook, JobGroupsSheetName, jobGroups, TableTheme);
+        }
+
+        private void PopulateAllDelegatesSheet(
+            IXLWorkbook workbook,
+            int centreId,
+            string? searchString,
+            string? sortBy,
+            string sortDirection,
+            string? filterString
+        )
+        {
+            var registrationPrompts = centreRegistrationPromptsService.GetCentreRegistrationPromptsByCentreId(centreId);
+            var delegatesToExport = GetDelegatesToExport(centreId, searchString, sortBy, sortDirection, filterString)
+                .ToList();
+
+            var dataTable = new DataTable();
+            // TODO: HEEDLS-810 - display extra PRN details if necessary. 

```

Comment:
Have you found out if the PRN will be necessary?

Reply:
Yep, added it. Uses the same wording as we do when we display it as requested by Kevin.